tests/ci_tests/data_driven/2/url

```diff
@@ -0,0 +1 @@
+http://127.0.0.1:5000/annotate/?text=The+paitient+was+diagnosed+with+both+cardiac+disease+and+renal+cancer.&model=MONDO
```

Comment:
Fixed by commit `ba756b0`. Data-driven test results are also updated as `start` and `end` values have changed. :heavy_check_mark: This will be fixed in other (non-CI) places in a different PR.